base_import_async/data/queue_job_function_data.xml

```diff
@@ -1,7 +1,13 @@
 <odoo noupdate="1">
+    <record model="queue.job.channel" id="channel_base_import">
+        <field name="name">base_import</field>
+        <field name="parent_id" ref="queue_job.channel_root" />
+    </record>
+
     <record id="job_function_base_import_import_split_file" model="queue.job.function">
         <field name="model_id" ref="base_import.model_base_import_import" />
         <field name="method">_split_file</field>
+        <field name="channel_id" ref="channel_base_import" />
```

Comment:
to make these changes effective we need a migration step IMO

Reply:
It can be changed by users, could be good to have a migration (only if no other channel than `root` was configured manually), but is not strictly required IMO